### PR TITLE
Add Binance isAlive implementation

### PR DESCRIPTION
**Pull request by roeiba**
This is done using same logic of Gemini who also uses ProductStreamingService

### CODE REVIEW

File: xchange-stream-core/pom.xml [hunk 1]
```diff
@@ -15,6 +15,12 @@
             <artifactId>service-core</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        
+        <dependency>
```
Comment (dozd) [inline xchange-stream-core/pom.xml]: this should not be in core as dependency. These services are only for exchange implementations. Why it is here?

Comment (roeiba) [reply xchange-stream-core/pom.xml]: POM file was pushed by mistake. Please ignore it.

**Comment by dozd** [comment]
Pls rebase onto develop

**Comment by dozd** [comment]
No response from the user. Closing as not merged.

**Pull request closed without merging**